subprojects/docs/src/docs/release/notes.md

```diff
@@ -81,6 +78,12 @@ TODO - Java lambdas are supported, and unsupported captured values are reported.
 TODO - File collections queried at configuration time are treated as configuration inputs.
 TODO - File system repositories are fully supported including dynamic versions in Maven, Maven local, and Ivy repositories
 
+### Convenient declaration of dependencies on Gradle plugins
+
+The dependency on a Gradle plugin can now be declared using either the plugin id or the plugin coordinates from a version catalog. 
+
+More details can be found in the [Plugins](userguide/platforms.html#sec:plugins) section and in the section on [Gradle Plugin Dependencies](userguide/declaring-dependencies.html#sec:gradle-plugin-dependencies).
```

Comment:
I think a single link to the new section in which we explain how to depend on plugins would be better. From there the manual has links to the other sections.

```suggestion
### Convenient declaration of dependencies on Gradle plugins

The dependency on a Gradle plugin can now be declared using either the plugin id or the plugin entry from a version catalog. 

More details can be found in the dedicated section of the [Implementing Gradle Plugins](implementing_gradle_plugins.html#[reacting_to_plugins](sec:depending_on_plugins)) user manual chapter.
```